pkg/dynamic/client_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	dynamicfake "k8s.io/client-go/dynamic/fake"
 	eventingv1beta1 "knative.dev/eventing/pkg/apis/eventing/v1beta1"
+	messagingv1beta1 "knative.dev/eventing/pkg/apis/messaging/v1beta1"
```

Comment:
I wonder whether we should already go to `v1` API (and then change eventing also to v1 in a subsequent PR).

Reply:
lets keep it at v1beta1 to also support working with earlier eventing installs?